ais_service_discovery/call_service.py

```diff
@@ -32,6 +37,12 @@ def run_service(service, body, opts={}):
             'Message': dumps(body),
             **opts
         })
+    if type in ['queue']:
```

Comment:
Can you update to work on both `queue` and `sqs` keyword just like the [JS lib](https://github.com/peak-ai/ais-service-discovery-js/blob/master/src/call-service.js#L62-L63)